### ISSUE TITLE
[APP SUBMITTED]: 'str' object does not support item assignment

### ISSUE THREAD

**Issue by TiagoDM** [SiCKRAGETV/sickrage-issues#790]
### INFO
Python Version: **2.7.8 (default, Jun 30 2014, 16:08:48) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-8-6.2.9200**
Branch: **develop**
Commit: SiCKRAGETV/SickRage@e3bd4dba929146619f3325c14a4cb0f0a4c60ca8
### ERROR
```
Thread-109 :: Failed doing webui callback: Traceback (most recent call last):
  File "C:\Users\Tiago\Desktop\SickRage-dev\sickbeard\webserve.py", line 231, in async_call
    result = function(**kwargs)
  File "C:\Users\Tiago\Desktop\SickRage-dev\sickbeard\webserve.py", line 2808, in downloadSubtitleMissed
    subtitles = show.getEpisode(int(season), int(episode)).downloadSubtitles()
  File "C:\Users\Tiago\Desktop\SickRage-dev\sickbeard\tv.py", line 1442, in downloadSubtitles
    notifiers.notify_subtitle_download(self.prettyName(), subtitleList)
  File "C:\Users\Tiago\Desktop\SickRage-dev\sickbeard\notifiers\__init__.py", line 101, in notify_subtitle_download
    n.notify_subtitle_download(ep_name, lang)
  File "C:\Users\Tiago\Desktop\SickRage-dev\sickbeard\notifiers\emailnotify.py", line 156, in notify_subtitle_download
    msg['Subject'] = lang + ' Subtitle Downloaded: ' + ep_name
TypeError: 'str' object does not support item assignment

```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by abeloin** [comment]
@TiagoDM Can you update PR merged in develop.

**Comment by fernandog** [comment]
fixed in develop

**Comment by TiagoDM** [comment]
ok. done. thank you

2015-03-04 0:52 GMT+00:00 Fernando <notifications@github.com>:

> Closed #790 <https://github.com/SiCKRAGETV/sickrage-issues/issues/790>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/sickrage-issues/issues/790#event-245673343>
> .
>



-- 
________________________________________
Tiago Marques
